ops/ansible/roles/bfd-server/templates/bfd-server.sh.j2

```diff
@@ -92,15 +89,16 @@ check_endpoint() {
 }
 
 ##
-# Runs test queries to verify that the server is working as expected. If not, issues a 'kill -INT' on the server's process to shut it down.
+# Runs test queries to verify that the server is working as expected before allowing traffic in the firewall.
 ##
-stop_service_if_failing() {
+
+service_startup_check() {
   # Reject all external traffic to service port until the application has started successfully
   sudo iptables -A INPUT -p tcp ! -i lo --dport "$BFD_PORT" -j REJECT
   log "Waiting $STARTUP_TESTING_BOOT_TIMEOUT seconds for initial startup"
   sleep $STARTUP_TESTING_BOOT_TIMEOUT
```

Comment:
Maybe this was just a bad conflict resolution? In any case, with these lines (but **without** `$STARTUP_TESTING_BOOT_TIMEOUT`), we're greeted with:
> 2022-06-23 22:05:29: Waiting  seconds for initial startup
> sleep: missing operand
> Try 'sleep --help' for more information.

Reply:
I set up this to 30 seconds, after testing it was enough for booting up